components/browse/ContentCardSection.vue

```diff
@@ -19,13 +19,16 @@
       data-qa="section group"
     >
       <ContentCard
-        v-for="card in section.fields.hasPart"
+        v-for="card in cards"
         :key="card.sys.id"
         :name="card.fields.name"
         :description="card.fields.description"
         :url="card.fields.url"
-        :image-url="card.fields.image.fields.file.url"
-        :image-title="card.fields.image.fields.title"
+        :image="card.fields.image"
+        :creator="card.fields.creator"
+        :institution="card.fields.provider"
```

Comment:
Unless there is a good reason not to, reuse the name of the property from `card.fields`, i.e. `:provider`. Same for the following two attributes.

Reply:
I have renamed these.